### ISSUE TITLE
`helmwave build` ensure CRDs are installed first

### ISSUE THREAD

**Issue by violinorg** [helmwave/helmwave#552]
** General info **

OS: linux
Helmwave Version 0.25.0
Environment: container ghcr.io/helmwave/helmwave:0.25.0 in gitlab-ci

**To Reproduce**

My helmwafe.yml.tpl
```
# Reserved for future
project: test-infra
version: 0.1.0

# Include repositories from file "helmwave-repositories.yml"
repositories:
  - name: jetstack
    url: https://charts.jetstack.io
  - name: scylla
    url: https://scylla-operator-charts.storage.googleapis.com/stable
  - name: nfs-subdir-external-provisioner
    url: https://kubernetes-sigs.github.io/nfs-subdir-external-provisioner/

# Some common option (uses YAML anchor)
.options: &options
  atomic: false
  create_namespace: true
  force: false
  max_history: 3
  timeout: 10m
  strict: true
  skip_crds: false
  wait: true
  wait_for_jobs: true

releases:

  - name: cert-manager
    chart:
      name: jetstack/cert-manager
      version: v1.11.0
    namespace: cert-manager
    tags:
      - cert-manager
    values:
      - values/_.yml
      - values/common/cert-manager/default.yml
      - values/common/cert-manager/{{ requiredEnv "ENVIRONMENT_NAME" }}.yml
    <<: *options

  - name: nfs-subdir-external-provisioner
    chart:
      name: nfs-subdir-external-provisioner/nfs-subdir-external-provisioner
      version: 4.0.16
    namespace: dbp-infra
    tags:
      - nfs
    values:
      - values/_.yml
      - values/common/nfs-subdir-external-provisioner/default.yml
      - values/common/nfs-subdir-external-provisioner/{{ requiredEnv "ENVIRONMENT_NAME" }}.yml
    <<: *options

  - name: scylla-operator
    chart:
      name: scylla/scylla-operator
      version: v1.7.4
    namespace: scylla-operator
    depends_on:
      - cert-manager@cert-manager
    tags:
      - scylla-operator
    values:
      - values/_.yml
      - values/common/scylla-operator/default.yml
      - values/common/scylla-operator/{{ requiredEnv "ENVIRONMENT_NAME" }}.yml
    <<: *options

  - name: scylla-manager
    chart:
      name: scylla/scylla-manager
      version: v1.7.4
    namespace: scylla-manager
    depends_on:
      - scylla-operator@scylla-operator
      - nfs-subdir-external-provisioner@dbp-infra
    tags:
      - scylla-manager
    values:
      - values/_.yml
      - values/common/scylla-manager/default.yml
      - values/common/scylla-manager/{{ requiredEnv "ENVIRONMENT_NAME" }}.yml
    <<: *options

  - name: scylla
    chart:
      name: scylla/scylla
      version: v1.7.4
    namespace: scylla
    depends_on:
      - scylla-operator@scylla-operator
      - scylla-manager@scylla-manager
      - nfs-subdir-external-provisioner@dbp-infra
    tags:
      - scylla
    values:
      - values/_.yml
      - values/common/scylla/default.yml
      - values/common/scylla/{{ requiredEnv "ENVIRONMENT_NAME" }}.yml
    <<: *options

```

<img width="1278" alt="Снимок экрана 2023-02-07 в 16 49 01" src="https://user-images.githubusercontent.com/60527410/217263524-b3f06a8c-a8a3-49a2-8bc8-b65c7dc82e80.png">

Help me bypass or fix this behavior

**Comment by violinorg** [comment]
Some of the errors have been fixed, there was a problem in the directories, I changed the structure, but did not update the paths to the variables

**Comment by violinorg** [comment]
I don't know what to do with CRD for cert-manager. 
<img width="1287" alt="Снимок экрана 2023-02-08 в 15 26 59" src="https://user-images.githubusercontent.com/60527410/217529814-4b82bd84-4440-4b45-9fb5-2e2ded44e304.png">

To install the certificate manager, there is a variable installCRDs: "true"
```
installCRDs: "true"

resources:
  limits:
    cpu: 100m
    memory: 256Mi
  requests:
    cpu: 10m
    memory: 32Mi

securityContext:
  fsGroup: 1000
  runAsUser: 1000
  runAsNonRoot: true

containerSecurityContext:
  capabilities:
    drop:
    - ALL

prometheus:
  enabled: false

affinity:
  nodeAffinity:
    preferredDuringSchedulingIgnoredDuringExecution:
    - weight: 1
      preference:
        matchExpressions:
        - key: node-role.kubernetes.io/master
          operator: Exists

tolerations:
    - key: node-role.kubernetes.io/master
      effect: NoSchedule
    - key: node-role.kubernetes.io/master
      effect: NoExecute

webhook:
  securityContext:
    fsGroup: 1000
    runAsUser: 1000
    runAsNonRoot: true

  containerSecurityContext:
    capabilities:
      drop:
      - ALL

  resources:
    limits:
      cpu: 100m
      memory: 256Mi
    requests:
      cpu: 10m
      memory: 32Mi

  affinity:
    nodeAffinity:
      preferredDuringSchedulingIgnoredDuringExecution:
      - weight: 1
        preference:
          matchExpressions:
          - key: node-role.kubernetes.io/master
            operator: Exists

  tolerations:
    - key: node-role.kubernetes.io/master
      effect: NoSchedule
    - key: node-role.kubernetes.io/master
      effect: NoExecute

cainjector:
  securityContext:
    fsGroup: 1000
    runAsUser: 1000
    runAsNonRoot: true

  containerSecurityContext:
    capabilities:
      drop:
      - ALL

  resources:
    limits:
      cpu: 100m
      memory: 256Mi
    requests:
      cpu: 10m
      memory: 32Mi

  affinity:
    nodeAffinity:
      preferredDuringSchedulingIgnoredDuringExecution:
      - weight: 1
        preference:
          matchExpressions:
          - key: node-role.kubernetes.io/master
            operator: Exists

  tolerations:
    - key: node-role.kubernetes.io/master
      effect: NoSchedule
    - key: node-role.kubernetes.io/master
      effect: NoExecute

```

**Comment by violinorg** [comment]
Very similar to https://github.com/helmwave/helmwave/issues/548

**Comment by violinorg** [comment]
If you start deployment by the cert-manager tag, then everything is fine
<img width="1309" alt="Снимок экрана 2023-02-09 в 16 57 01" src="https://user-images.githubusercontent.com/60527410/217832958-c5cdac1e-e200-460d-b5d7-8a9490d88436.png">

**Comment by zhilyaev** [comment]
will fix in #611
